configuration/vpn.md

```diff
@@ -60,67 +56,40 @@ Using a ProxyVM to set up a VPN client gives you the ability to:
 
 5. Optionally, you can install some [custom icons](https://github.com/Zrubi/qubes-artwork-proxy-vpn) for your VPN
 
-#### Using iptables and openvpn
 
-1. Create a new VM and check the ProxyVM radio button.
+Set up a ProxyVM as a VPN gateway using iptables and CLI scripts
+----------------------------------------------------------------
 
-    ![Create\_New\_VM.png](/attachment/wiki/VPN/Create_New_VM.png)
-    
-    If your choice of template VM doesn't already have the `openvpn` package, you'll need to install it in the template first. You may also need to `systemctl disable` any openvpn service that comes with the package if you follow the instructions for autostart below.
+This method is more involved than the one above, but has anti-leak features that also make the connection _fail closed_ should it be interrupted. It has been tested with Fedora 23 and Debian 8 templates.
 
-2.  Set up OpenVPN.
+1. Create a new VM: Name it, click the ProxyVM radio button then choose a color and template.
 
-    Copy your openvpn config files to `/rw/config/openvpn/` folder. The example main config file is `openvpn-client.ovpn`.
-
-    It should have one line that reads `dev tun`.
-
-    If it does not contain a line `redirect-gateway def1` you may wish to add it. This will route all traffic through your vpn's network device after a connection is created. However, many VPN services will push this instruction to your client automatically -- having a line that says `client` or `pull` in your openvpn config instructs your client to use parameters specified by the VPN server.
+    ![Create\_New\_VM.png](/attachment/wiki/VPN/Create_New_VM.png)
     
-    NOTE: If the connection breaks down all traffic will by default be routed through the upstream network device eth0 (we will stop this with iptables in step 3).
-
-    Also add the following to accomodate a DNS script:
-
-    ~~~
-    script-security 2
-    up 'qubes-vpn-handler.sh up'
-    down 'qubes-vpn-handler.sh down'
-    ~~~
-
-3.  Set up iptables.
-
-    Edit the firewall script with `sudo nano /rw/config/qubes-firewall-user-script` and add:
-
-	~~~
-	#!/bin/bash
-	#    First, block all outgoing traffic
-	iptables -P OUTPUT DROP
-    iptables -F OUTPUT
-
-	#    Add the `qvpn` group to system, if it doesn't already exist
-    if ! grep -q "^qvpn:" /etc/group ; then
-        groupadd -rf qvpn
-        sync
-    fi
-    sleep 2s
-
-	#    Allow traffic from the `qvpn` group to the uplink interface (eth0);
-	#    Our openvpn will run as group `qvpn`.
-    iptables -A OUTPUT -p all -o eth0 -m owner --gid-owner qvpn \
-    -m state --state NEW,ESTABLISHED -j ACCEPT
-
-	#	Allow internal system connections:
-    iptables -I OUTPUT -o lo -j ACCEPT
+    Note: Do not enable NetworkManager, as it can interfere with the scripts' DNS features. If you used NetworkManager or other method in a previous attempt, do not re-use the old ProxyVM... Create a new one according to this step.
```

Comment:
Since enabling NetworkManager in ProxyVM is not obvious option, this can be confused with using NetworkManager in NetVM. Which is obviously not what this note is about.